models/public/efficientnet-b5-pt/model.yml

```diff
@@ -0,0 +1,79 @@
+# limitations under the License.
+
+description: >-
+  The `efficientnet-b5-pt` model is one of the efficientnet <https://arxiv.org/abs/1905.11946>
+  group of models designed to perform image classification. This model was pretrained
+  in TensorFlow\*, then weights were converted to PyTorch\* All the EfficientNet models
+  have been pretrained on the ImageNet image database. For details about this family
+  of models, check out the repository <https://github.com/rwightman/gen-efficientnet-pytorch>.
+
+  The model input is a blob that consists of a single image of 1x3x456x456 in RGB
+  order. The BGR mean values need to be subtracted as follows: [123.675,116.28,103.53]
+  before passing the image blob into the network. In addition, values must be multiplied
+  by [58.395,57.12,57.375].
+
+  The model output for `efficientnet-b5-pt` is the typical object classifier output
+  for the 1000 different classifications matching those in the ImageNet database.
+task_type: classification
+files:
+  - name: model/gen_efficientnet.py
+    size: 40997
+    sha256: 8613d48ca74611d3566d8c02cbf7c92aad6b16a708ffcf21183a7014fecdec09
+    source: https://raw.githubusercontent.com/rwightman/gen-efficientnet-pytorch/master/gen_efficientnet/gen_efficientnet.py
+  - name: model/efficientnet_builder.py
+    size: 18446
+    sha256: 69bb2adc49dc79c8860f36acca910bad6733e23b46ed80551248b90141d5e1b5
+    source: https://raw.githubusercontent.com/rwightman/gen-efficientnet-pytorch/master/gen_efficientnet/efficientnet_builder.py
+  - name: model/helpers.py
+    size: 1097
+    sha256: 0415f198b8a87cb34c9f9aed79f267043010fbd197e27df805bab9d070da82ed
+    source: https://raw.githubusercontent.com/rwightman/gen-efficientnet-pytorch/master/gen_efficientnet/helpers.py
+  - name: model/conv2d_helpers.py
+    size: 6175
+    sha256: e6ba7e878dd28c6d0ccc9707205bf6b58bea664d7fee820d7101a6a8e48990a9
+    source: https://raw.githubusercontent.com/rwightman/gen-efficientnet-pytorch/master/gen_efficientnet/conv2d_helpers.py
+  - name: model/__init__.py
+    size: 32
+    sha256: e0bedda5f5e949b8ace7d4f5cdf80e7e664c0b0a935486b6152078fa61c80c1b
+    source: https://github.com/rwightman/gen-efficientnet-pytorch/raw/master/gen_efficientnet/__init__.py
+  - name: tf-efficientnet-b5.pth
+    size: 122398414
+    sha256: 99018a74e61e5948a955ebfaa2b02ba9abe7bb2e6b7f3d2dfe100e07e103bbdb
+    source: https://github.com/rwightman/pytorch-image-models/releases/download/v0.1-weights/tf_efficientnet_b5_aa-99018a74.pth
+postprocessing:
+  - $type: regex_replace
+    file: model/conv2d_helpers.py
+    pattern: '_EXPORTABLE = False'
+    replacement: '_EXPORTABLE = True'
+pytorch_to_onnx:
+  - --model-path=$dl_dir
+  - --model-name=tf_efficientnet_b5
+  - --import-module=model
+  - --weights=$dl_dir/tf-efficientnet-b5.pth
+  - --input-shape=1,3,456,456
+  - --input-names=data
+  - --output-names=prob
+  - --output-file=$conv_dir/efficientnet-b5.onnx
+  - --model-params=pretrained=False
```

Comment:
`pretrained` seems to already default to `False` for every model, so is this parameter (and the changes to the conversion script) even necessary?

Reply:
I see you have removed the `--model-params` arguments from all models, so maybe now you could remove the corresponding code from the converter?

Reply:
No, it will be useful in future.